src/presentation/http/router/note.test.ts

```diff
@@ -1,27 +1,38 @@
 
 import { describe, test, expect, beforeEach } from 'vitest';
-
-import notes from '@tests/test-data/notes.json';
-import noteSettings from '@tests/test-data/notes-settings.json';
-import noteTeams from '@tests/test-data/note-teams.json';
+import { MemberRole } from '../../../domain/entities/team';
 
 describe('Note API', () => {
+  beforeEach(async () => {
+    await global.db.truncateTables();
+  });
```

Comment:
can you delete [these lines](https://github.com/codex-team/notes.api/blob/main/src/presentation/http/router/note.test.ts#L423-L440)?

They are not needed anymore